README.md

```diff
@@ -286,6 +137,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
Great job @AjroudiRida 🤝

- [ ] Kindly fix the broken MIT license link to make your readme file more professional.
![MIT license](https://github.com/AjroudiRida/vet_clinic/assets/63917725/9522d4cd-26a7-4c4a-a873-3d1330bed8bc)